cmd/expiration-mailer/main.go

```diff
@@ -641,42 +641,45 @@ func (ds durationSlice) Swap(a, b int) {
 
 type Config struct {
 	Mailer struct {
-		DebugAddr string
+		DebugAddr string `validate:"required,hostname_port"`
 		DB        cmd.DBConfig
 		cmd.SMTPConfig
 
-		// From is the "From" address for reminder messages.
-		From string
+		// From is an RFC 5322 formatted "From" address for reminder messages,
+		// e.g. "Example <example@test.org>"
+		From string `validate:"required"`
 
-		// Subject is the Subject line of reminder messages.
-		// This is a Go template with a single variable: ExpirationSubject,
-		// which contains a list of affectd hostnames, possible truncated.
+		// Subject is the Subject line of reminder messages. This is a Go
+		// template with a single variable: ExpirationSubject, which contains
+		// a list of affected hostnames, possible truncated.
 		Subject string
 
 		// CertLimit is the maximum number of certificates to investigate in a
-		// single batch.
+		// single batch. Defaults to 100.
```

Comment:
min=0 to prevent negative integers?

Also applies to the two ints immediately below this one.